Library/Homebrew/download_strategy.rb

```diff
@@ -312,6 +323,11 @@ def _fetch
     curl @url, "-C", downloaded_size, "-o", temporary_path
   end
 
+  def actual_url(url)
+    m = Utils.popen_read("curl", "-I", "-L", url).match /^Location: (.+)$/
```

Comment:
@xu-cheng Your version causes some issues:

```
==> Downloading https://downloads.sourceforge.net/project/ccd2iso/ccd2iso/ccd2iso-0.3/ccd2iso-0.3.tar.gz
Error: undefined method `start_with?' for #<Array:0x007fa98a94f5f8>
```

I adjusted the syntax at the top to fit the new `actual_urls` language but it still grumped at me. @vszakats' version worked if people are happy with that?